roles/splunk/tasks/configure_idxc_manager.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Run splunk version command to check currently installed version
```

Comment:
Don't duplicate existing tasks. If you want to re-use a task, split it out and use `include_tasks` instead.

Reply:
Gotcha, thanks Mason - I'll work on splitting that task out.

Reply:
I'd recommend moving https://github.com/splunk/ansible-role-for-splunk/blob/master/roles/splunk/tasks/check_splunk.yml#L21 to a new task called `check_splunk_version.yml` then include that where needed.